### PR TITLE
WIP: Update NfcGuideView

### PR DESCRIPTION
**Pull request by Franzaine**
This PR:
- Adds new USB graphics
- Updates current graphics to bring them more in line

### CODE REVIEW

**Comment by Franzaine** [comment]
Not done yet, but soon..

File: android/src/main/java/nordpol/ui/NfcGuideView.java [hunk 1]
```diff
@@ -100,8 +104,11 @@ public NfcGuideView(Context context, AttributeSet attrs) {
         case 1:
             mNfcGuideHand.setImageDrawable(getResDrawable(R.drawable.nfc_guide_view_hand_holding_card_black));
             break;
-	case 2:
-	    mNfcGuideHand.setImageDrawable(getResDrawable(R.drawable.nfc_guide_view_hand_holding_usb_black));
+	    case 2:
```
Comment (pepegar) [inline android/src/main/java/nordpol/ui/NfcGuideView.java]: Seems like indentation here is broken

File: android/src/main/java/nordpol/ui/NfcGuideView.java [hunk 2]
```diff
@@ -31,6 +31,8 @@
  * To choose between the different NFC device graphics add the following line:
  * For a black card: app:nfc_device="card_black"
  * For a ruby card: app:nfc_device="card_ruby"
+ * For a black USB: app:nfc_device="usb_black"
+ * For a black USB with a Fidesmo logo: app:nfc_device="usb_black_fidesmo"
```
Comment (Franzaine) [inline android/src/main/java/nordpol/ui/NfcGuideView.java]: Documentation update

File: android/src/main/java/nordpol/ui/NfcGuideView.java [hunk 3]
```diff
@@ -70,6 +72,8 @@
 
     public NfcGuideView(Context context, AttributeSet attrs) {
         super(context, attrs);
+        
+        guideItemsStartDistance = getResources().getInteger(R.integer.GUIDE_ITEMS_START_DISTANCE);
```
Comment (Franzaine) [inline android/src/main/java/nordpol/ui/NfcGuideView.java]: "guideItemsStartDistance" is now saved as a resource to make Android automatically choose the version most fitting for the current screen resolution. This makes the distance between the two devices in the NfcGuideView more alike no matter what screen resolution you have.

File: android/src/main/java/nordpol/ui/NfcGuideView.java [hunk 4]
```diff
@@ -44,7 +46,7 @@
  * nfcGuideView.setCurrentStatus(NfcGuideView.NfcGuideViewStatus.TRANSFERRING);
  */
 public class NfcGuideView extends RelativeLayout {
-    private static final int GUIDE_ITEMS_START_DISTANCE = 100;
+    private int guideItemsStartDistance = 50;
```
Comment (Franzaine) [inline android/src/main/java/nordpol/ui/NfcGuideView.java]: Just setting a functional default, this is replaced with whatever is in the resources later

File: android/src/main/java/nordpol/ui/NfcGuideView.java [hunk 5]
```diff
@@ -100,8 +104,11 @@ public NfcGuideView(Context context, AttributeSet attrs) {
         case 1:
             mNfcGuideHand.setImageDrawable(getResDrawable(R.drawable.nfc_guide_view_hand_holding_card_black));
             break;
-	case 2:
-	    mNfcGuideHand.setImageDrawable(getResDrawable(R.drawable.nfc_guide_view_hand_holding_usb_black));
+	    case 2:
+	        mNfcGuideHand.setImageDrawable(getResDrawable(R.drawable.nfc_guide_view_hand_holding_usb_black));
+            break;
+	    case 3:
+	        mNfcGuideHand.setImageDrawable(getResDrawable(R.drawable.nfc_guide_view_hand_holding_usb_black_fidesmo));
```
Comment (Franzaine) [inline android/src/main/java/nordpol/ui/NfcGuideView.java]: Added a USB device with our logo

**Comment by Franzaine** [comment]
@pepegar ready for review!
Note: The SVG files are the main part of the diff. That is why it is so big. They are there because they are open source together with the library.

**Comment by pepegar** [comment]
+1, just fix the indentation ;)

**Pull request merged**